package/.eslintrc.cjs

```diff
@@ -0,0 +1,61 @@
+// eslint-disable-next-line no-undef
+module.exports = {
+    env: {
+        browser: true,
+        es2021: true
+    },
+    extends: [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended"
+    ],
+    overrides: [
+    ],
+    parser: "@typescript-eslint/parser",
+    parserOptions: {
+        ecmaVersion: "latest",
+        sourceType: "module"
+    },
+    plugins: [
+        "@typescript-eslint"
+    ],
+    ignorePatterns: [
+        "**/*.js",
+        "**/*.d.ts"
+    ],
+    rules: {
+        indent: [
+            "error",
+            4
+        ],
+        "linebreak-style": [
+            "error",
+            "unix"
+        ],
+        quotes: [
+            "error",
+            "double"
+        ],
+        semi: "off",
+        "@typescript-eslint/semi": ["error", "never"],
+        "@typescript-eslint/member-delimiter-style": [
+            "error",
+            {
+                multiline: {
+                    delimiter: "none",
+                    requireLast: false
+                },
+                singleline: {
+                    delimiter: "comma",
+                    requireLast: false
+                }
```

Comment:
I would suggest semicolon line ending as the google style guide suggests.

Reply:
Personally, I believe that if a language has optional semicolons, that they should be used, but I'll change the eslint config

Reply:
Don't understand your comment. You also suggest to use semicolons? But not using them anywhere?